src/lib/swapper/swappers/CowSwapper/cowExecuteTrade/cowExecuteTrade.test.ts

```diff
@@ -123,6 +135,31 @@ const cowTradeFoxToEth: CowTrade<KnownChainIds.EthereumMainnet> = {
   minimumBuyAmountAfterFeesCryptoBaseUnit: '46868859830863283',
 }
 
+const cowTradeUsdcToXdai: CowTrade<KnownChainIds.GnosisMainnet> = {
+  rate: '5.462e-17',
```

Comment:
Don't use scientific notation, all other tests use full-precision float strings for rate

Reply:
https://github.com/shapeshift/web/pull/4595#discussion_r1212912283